SIA.tex

```diff
@@ -472,7 +443,9 @@ \subsubsection{INSTRUMENT}
 The INSTRUMENT parameter is a string-valued parameter that specifies the name of the instrument with which the data was acquired. The value is compared with the instrument\_name from the ObsCore data model.
 
 \subsubsection{DPTYPE}
-The DPTYPE parameter is a string-valued parameter that specifies the type of data. The value is compared with the dataproduct\_type from the ObsCore data model. For the SIA \{query\} resource, the only values that should be returned for dataproduct\_type are \textit{image} and \textit{cube}, so this parameter can be only really be used to select one of these.
+The DPTYPE parameter is a string-valued parameter that specifies the type of data. The value is compared with the dataproduct\_type from the ObsCore data model.  In contrast to SIA2.0 which allows only dataproduct\_types  \textit{image} and \textit{cube}, all the other dataproduct\_types are allowed, in such a way that we can constrain the service to retrieve visibility data, timeseries and event lists. Using DPTYPE with the "spectrum" value can be considered as an upgrade of the SSA protocol.
```

Comment:
Should we include catalogs here? CASDA has downloadable catalogs included in obscore but the  dataproduct_type for those must be null according to obscore, which hinders discoverability.

Reply:
Tricky point. Please open a new issue for that